neps/nep-0171.md

```diff
@@ -218,21 +218,21 @@ function nft_on_transfer(
 
 ### Events
 
-NEAR and third-party applications need to track `mint`, `transfer`, `burn`, and `contract_metadata_update` events for all NFT-driven apps consistently.
+NEAR and third-party applications need to track `mint`, `update`, `transfer`, `burn`, and `contract_metadata_update` events for all NFT-driven apps consistently.
 This extension addresses that.
 
 Keep in mind that applications, including NEAR Wallet, could require implementing additional methods to display the NFTs correctly, such as [`nft_metadata`][Metadata] and [`nft_tokens_for_owner`][NFT Enumeration]).
 
 #### Events Interface
 
-Non-Fungible Token Events MUST have `standard` set to `"nep171"`, standard version set to `"1.1.0"`, `event` value is one of `nft_mint`, `nft_burn`, `nft_transfer`, or `contract_metadata_update`, and `data` must be of one of the following relevant types: `NftMintLog[] | NftTransferLog[] | NftBurnLog[] | NftContractMetadataUpdateLog[]`:
+Non-Fungible Token Events MUST have `standard` set to `"nep171"`, standard version set to `"1.2.0"`, `event` value is one of `nft_mint`, `nft_burn`, `nft_transfer`, `nft_metadata_update`, or `contract_metadata_update`, and `data` must be of one of the following relevant types: `NftMintLog[] | NftTransferLog[] | NftBurnLog[] | NftMetadataUpdateLog[] | NftContractMetadataUpdateLog[]`:
 
 ```ts
 interface NftEventLogData {
     standard: "nep171",
-    version: "1.1.0",
-    event: "nft_mint" | "nft_burn" | "nft_transfer" | "contract_metadata_update",
-    data: NftMintLog[] | NftTransferLog[] | NftBurnLog[] | NftContractMetadataUpdateLog[],
+    version: "1.2.0",
+    event: "nft_mint" | "nft_burn" | "nft_transfer" | "nft_metadata_update" | "contract_metadata_update",
```

Comment:
Side comment for the future events (not the NEP-171, as it is already set).
Personally, I don't think we need to have `nft_` prefix in all the event names (not the struct names... they are not relevant to the standard). The NFT namespace is already implicity with the `standard: "nep171"`.

Reply:
I disagree, I think this prefix makes it easier for indexers to handle events by type, and it helps categorize events within the contract.